Makefile

```diff
@@ -11,6 +11,11 @@ install:
 	mkdir -p -- "${DESTDIR}${MANPREFIX}/man/man1"
 	install -m 444 "${VIPE_SH_MAN}" "${DESTDIR}${MANPREFIX}/man/man1"
 
+.PHONY: uninstall
+uninstall:
+	$(RM) "${DESTDIR}${PREFIX}/bin/${VIPE_SH_CMD}"
```

Comment:
Let's replace `$(RM)` with `rm -f`:

- `RM` is not defined when using bmake. 

As a side note, this makefile is using curly brackets instead of parentheses for variables.  This is the preferred style in the communities using bmake as its make implementation, e.g., FreeBSD.

Reply:
Understood :)
Just added another commit and updated the file!